renderdoc/driver/d3d12/d3d12_device_wrap.cpp

```diff
@@ -728,26 +901,174 @@ HRESULT WrappedID3D12Device::CreateCommittedResource(const D3D12_HEAP_PROPERTIES
 
 HRESULT WrappedID3D12Device::CreateHeap(const D3D12_HEAP_DESC *pDesc, REFIID riid, void **ppvHeap)
 {
-  return m_pDevice->CreateHeap(pDesc, riid, ppvHeap);
+  if(ppvHeap == NULL)
+    return m_pDevice->CreateHeap(pDesc, riid, nullptr);
```

Comment:
Just use `NULL` here, not `nullptr`, also in the same place for in the other functions added.